nextcord/ext/commands/core.py

```diff
@@ -320,7 +320,8 @@ def __init__(
             raise TypeError("Name of a command must be a string.")
         self.name: str = name
 
-        self._callback = func
+        # pyright complains that ContextT is incompatible with plain Context
```

Comment:
Becuase it isn't. Consider fixing the issue or rewrite this comment to not seem so opposed to pyright.

Reply:
I don't know how to fix this problem, so I'll rephrase the comment.